src/lib/Permission/LimitationResolver.php

```diff
@@ -58,7 +58,7 @@ public function getContentCreateLimitations(Location $parentLocation): LookupLim
         $contentInfo = $parentLocation->getContentInfo();
         $contentType = $contentInfo->getContentType();
         $contentCreateStruct = $this->contentService->newContentCreateStruct($contentType, $contentInfo->getMainLanguageCode());
-        $contentCreateStruct->sectionId = $contentInfo->getSection();
+        $contentCreateStruct->sectionId = $contentInfo->getSection()->id;
```

Comment:
This will throw a permission error when it accesses the proxy object if the user does not have section view permissions, which would be odd in my opinion as this is the permission system checking , not the user viewing the section. 

Perhaps $contentCreateStruct->sectionId = $contentInfo->sectionId; is a better route?

Reply:
Indeed, this is a better approach. Should I update my PR?

Reply:
I would think so, I don't believe in this case the user should need section view as the permission system is using it internally and not actually letting the user see any section details.

Reply:
:+1: for `$contentInfo->sectionId` too!